etc/conf/atTelemetry.yaml

```diff
@@ -55,6 +52,7 @@ end_collection_event:
 imageParam_event:
     device: ATCamera
     topic: imageReadoutParameters
+    array_keys: ccdNames
```

Comment:
what's this??

Reply:
The key that maps to the arrays for CCD/Sensors. Here I only made `array_keys` visible at the configuration label.